test/unit/client-daemonset.bats

```diff
@@ -418,9 +418,205 @@ load _helpers
       --set 'global.gossipEncryption.secretName=bar' \
       . | tee /dev/stderr |
       yq '.spec.template.spec.containers[] | select(.name=="consul") | .command | join(" ") | contains("encrypt")' | tee /dev/stderr)
+  [ "${actual}" == "true" ]
+}
+
+#--------------------------------------------------------------------
+# global.tls.enabled
+
+@test "client/DaemonSet: no secret volumes when TLS is disabled" {
```

Comment:
seems fragile, what if I add a secret for something unrelated? can you test just that it doesn't have the tls volumes? Or... delete this test since checking the absence of things for each feature seems like overkill

Reply:
Yeah I think you're right. Deleted the test.